experiments/download_ssl4eo.py

```diff
@@ -519,52 +449,28 @@ def worker(idx: int) -> None:
         if str(idx) in ext_coords.keys():
             return
 
-        (patches_s1, patches_s2c, patches_s2a, center_coord) = get_random_patches_match(
+        patches, center_coord = get_random_patches_match(
             idx,
-            collections,
+            collection,
             bands,
             crops,
+            dtype,
             dates,
-            radius=radius,
+            radius=args.radius,
             debug=args.debug,
             match_coords=match_coords,
         )
 
-        if (
-            patches_s1 is not None
-            and patches_s2c is not None
-            and patches_s2a is not None
-        ):
+        if patches is not None:
             if args.save_path is not None:
```

Comment:
I don't think it's possible for this to be None

Reply:
it is possible, see line 251-254:

```
except (ee.EEException, urllib3.exceptions.HTTPError) as e:
        if debug:
            print(e)
        return None, coords
```

Reply:
It's possible for patches, but not for save_path